Evaluator/TA/ai_evaluator/ai.py

```diff
@@ -43,7 +43,6 @@ class GPTEvaluator(evaluators.TAEvaluator):
         "Kaufman Adaptive Moving Average": tulipy.kama,
         "Hull Moving Average": tulipy.kama,
         "RSI: Relative Strength Index": tulipy.rsi,
-        "Stochastic RSI": tulipy.stochrsi,
```

Comment:
😲 it's not working ?

Reply:
it's giving results that are not allowing to forcast a trend